config/config_hmpxv1.yaml

```diff
@@ -15,21 +15,55 @@ strain_id_field: "accession"
 display_strain_field: "strain"
 
 build_name: "hmpxv1"
-auspice_name: "monkeypox_hmpxv1"
+auspice_name: "mpox_mpxv-IIb"
 
 ## filter
 min_date: 2017
 min_length: 100000
-sequences_per_group: "--sequences-per-group 40"
-group_by: "--group-by year month country"
-filters: "--exclude-where outbreak!=hMPXV-1"
+
+
+### Set 1: Non-B.1 sequences: use all
+### Set 2: B.1 sequences: small sample across year/country, maybe month
+filter:
+  non_b1:
+    group_by: "--group-by lineage year country"
+    sequences_per_group: "--sequences-per-group 50"
+    other_filters: "outbreak!=hMPXV-1 clade!=IIb"
```

Comment:
CI is failing because the example data uses old clades. Can you [update](https://github.com/nextstrain/monkeypox/blob/cb7701f7acbc7fd35e98bb6c06668329b30ff870/README.md?plain=1#L121-L126)?

Reply:
Done, thanks!